notebooks/icrp107_dataset/icrp107_dataset.ipynb

```diff
@@ -314,6 +405,12 @@
     "    add[\"Z\"], add[\"A\"] = get_Z_A(add[\"Radionuclide\"])\n",
     "    if add[\"Radionuclide\"][-1].isalpha():\n",
     "        add[\"Metastable_state\"] = add[\"Radionuclide\"][-1]\n",
+    "    if (line[0].endswith(\"m\") == True\n",
+    "        or line[0].endswith(\"n\") == True):\n",
```

Comment:
This if statement isn't needed, I think. You can just incorporate the next statement into the if block above, as that also only executes for the metastable isomers.